src/crypto/store/base.js

```diff
@@ -9,3 +9,23 @@
  *
  * @interface CryptoStore
  */
+
+/**
+ * Represents an outgoing room key request
+ *
+ * @typedef {Object} OutgoingRoomKeyRequest
+ *
+ * @property {string} requestId    unique id for this request
+ *
+ * @property {string?} cancellationRequestId
+ *    transaction id for the cancellation, if any
```

Comment:
It might be worth mentioning that on `:18` then when describing what `requestId` is then, since I didn't manage to grok that from looking at the code alone.